src/Indigo/Contracts/HomebaseLite/Impl/Admin.hs

```diff
@@ -51,8 +51,8 @@ acceptAdmin _ = do
   setStorageField @Storage #sAdmin candidate
 
 -- TODO [morley#748] should use senderIsNotAdmin, but can't because Lorentz exprots it already
-[errorDoc| "senderIsNotAdmin_" exception "The sender is not the current admin"|]
-[errorDoc| "senderIsNotAdminCandidate" exception "The sender is not the current admin candidate"|]
+[errorDocArg| "senderIsNotAdmin_" exception "The sender is not the current admin"|]
```

Comment:
morley#748 was fixed a while ago, so you could rename this error to `senderIsNotAdmin`, if you're willing to break compatibility a wee little bit (I don't think there's anything we need to maintain compatibility with yet?)

Reply:
Changed